ajaxorg/ace
# Update Jshint to 2.13.5

*Issue #, if available:* #4609

*Description of changes:*
Update Jshint to 2.13.5

To update `jshint` to new version:
1. Clone last version from https://github.com/jshint/jshint
2. Replace all `lodash` with `underscore` in requires in `jshint/src/*.js`
3. Add `_.slice = require("lodash.slice");` to `src/scope-manager.js` after `var _ = ...`
4. Add `_.clone = require("lodash.clone");` to `src/jshint.js` after `var _ = ...`
5. Add ```  
   "underscore": latest
   "lodash.clone": "^4.5.0",
   "lodash.slice": "^4.2.0"
   ``` to `package.json`, remove `lodash` and run `npm i`
6. Change in `ace/tool/update_deps.js` jshint path `deps.jshint.browserify.path` with your path to changed jshint
7. Run `node update_deps.js jshint`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
